AdminPortal/src/pages/admin.tsx

```diff
@@ -60,12 +74,13 @@ export default function Admin() {
               </tr>
             </thead>
             <tbody className={styles.tablebody}>
+
               {data.map((item) => (
-                <tr key={JSON.stringify(item)}>
-                  <td>{item.name}</td>
-                  <td>{item.phone}</td>
-                  <td>{item.role}</td>
-                </tr>
+                  <tr key={JSON.stringify(item)} onClick={() => handleRowClick(item)}>
```

Comment:
I think we can have the key here be something besides the stringified item. Something that'll be unique to an admin user, like an email (or the phone here would also work)

Reply:
have made it the phone number right now. Maybe once we have the actual data in mongoDB or something we can change it to the id or email. Their is no email attribute at the moment.